v21.2/create-table.md

```diff
@@ -111,6 +111,51 @@ Parameter | Description
 `ON COMMIT PRESERVE ROWS` | This clause is a no-op, allowed by the parser for PostgresSQL compatibility. CockroachDB only supports session-scoped [temporary tables](temporary-tables.html), and does not support the clauses `ON COMMIT DELETE ROWS` and `ON COMMIT DROP`, which are used to define transaction-scoped temporary tables in PostgreSQL.
 `opt_interleave` | {% include {{ page.version.version }}/misc/interleave-deprecation-note.md %}
 
+## Column qualifications
+
+CockroachDB supports the following column qualifications:
+
+- [Column-level constraints](constraints.html)
+- [Collations](collate.html)
+- [Column family assignments](column-families.html)
+- [`DEFAULT` expressions](default-value.html)
+- <span class="version-tag">New in v21.2</span>: [`ON UPDATE` expressions](#on-update-expressions)
+- <span class="version-tag">New in v21.2</span>: [Identity columns](#identity-columns) (sequence-populated columns)
+
+### `ON UPDATE` expressions
+
+<span class="version-tag">New in v21.2</span>: `ON UPDATE` expressions update column values in the following cases:
+
+- An [`UPDATE`](update.html) or [`UPSERT`](upsert.html) statement modifies a different column value in the same row.
+- An `ON UPDATE CASCADE` [foreign key action](foreign-key.html#foreign-key-actions) on a different column modifies an existing value in the same row.
```

Comment:
The different structure of these two bullets make it harder to understand the behavior in the second case. Could the second one be rewritten as follows:

An `ON UPDATE CASCADE` [foreign key action](foreign-key.html#foreign-key-actions) modifies a different column value in the same row.

Reply:
Done.